inc/search.class.php

```diff
@@ -3102,129 +3147,169 @@ static function addHaving($LINK, $NOT, $itemtype, $ID, $searchtype, $val) {
     * Generic Function to add ORDER BY to a request
     *
     * @since 9.4: $key param has been dropped
+    * @since 10.0.0: Parameters changed to allow multiple sort fields.
+    *    Old functionality maintained by checking the type of the first parameter.
+    *    This backwards compatibility will be removed in a later version.
     *
-    * @param string  $itemtype  ID of the device type
-    * @param integer $ID        field to add
-    * @param string  $order     order define
+    * @param array  $sort_fields The search options to order on. This array should contain one or more associative arrays containing:
+    *    - itemtype: The itemtype the search option belongs to
+    *    - id: The search option ID
+    *    - order: The sort direction (Default: ASC). Invalid sort directions will be replaced with the default option
+    * @param ?integer $_id    field to add (Deprecated)
+    * @param string  $_order  order define (Deprecated)
     *
-    * @return select string
+    * @return string ORDER BY query string
     *
    **/
-   static function addOrderBy($itemtype, $ID, $order) {
+   static function addOrderBy($sort_fields, $_id = null, $_order = 'ASC') {
```

Comment:
You should add an entry in CHANGELOG for this. Some plugins are using this method (`fusioninventory` for instance).